vm/systemSmartContracts/delegation.go

```diff
@@ -1226,6 +1232,19 @@ func (d *delegation) stakeNodes(args *vmcommon.ContractCallInput) vmcommon.Retur
 	return vmcommon.Ok
 }
 
+func tooManyNodesLogs(logEntries []*vmcommon.LogEntry) bool {
+	for _, logEntry := range logEntries {
+		if len(logEntry.Topics) != 1 {
+			continue
+		}
+		if !bytes.Equal(logEntry.Topics[0], []byte(numberOfNodesTooHigh)) {
```

Comment:
shouldn't this be `if bytes.Equal(logEntry.Topics[0], []byte(numberOfNodesTooHigh))`

Reply:
Right, fixed it